src/applications/discharge-wizard/components/gpSteps/StepTwo.jsx

```diff
@@ -37,9 +37,10 @@ const renderMedicalRecordInfo = formValues => {
             You can request your <strong>VA medical records</strong> by
             submitting VA Form 10-5345 to your local VA Medical Center.{' '}
             <a
+              download
               target="_blank"
```

Comment:
Do we need download + target="_blank"?

Reply:
Yeah, since it is va.gov domain, I can remove the target attribute.